userguide/system.rst

```diff
@@ -2403,13 +2403,12 @@ To generate a support ticket, fill in the fields:
 * **Attachments** is an optional field where configuration files or
   screenshots of any errors or tracebacks can be included.
 
-After completing the fields, click the :guilabel:`Submit` button to
-generate and send the support ticket to iXsystems. A pop-up menu
-provides a clickable URL to view the status of or add additional
-information to that support ticket.
-When not already logged into the
-`iXsystems Support page <https://support.ixsystems.com/>`__, clicking
-this URL prompts for a login, or to register a new login.
+Click :guilabel:`Submit` to generate and send the support ticket to
```

Comment:
By making the attachments paragraph TN-only, we can move the text from above down here and replace this.  This paragraph is not specific to either FN or TN, so we can factor it out like this.

Please come talk to me about this before doing it to make sure I'm not missing something.